### ISSUE TITLE
linter: jsx_a11y/no-noninteractive-element-to-interactive-role failed to resolve config

### ISSUE THREAD

**Issue by xsjcTony** [oxc-project/oxc#23476]
### What version of Oxlint are you using?

1.70.0

### What command did you run?

oxlint

### What does your `.oxlintrc.json` (or `oxlint.config.ts`) config file look like?

```ts
    'jsx-a11y/no-noninteractive-element-to-interactive-role': [
      'error',
      {
        ul: ['listbox', 'menu', 'menubar', 'radiogroup', 'tablist', 'tree', 'treegrid'],
        ol: ['listbox', 'menu', 'menubar', 'radiogroup', 'tablist', 'tree', 'treegrid'],
        li: ['menuitem', 'option', 'row', 'tab', 'treeitem'],
        table: ['grid'],
        td: ['gridcell'],
      },
    ],
```

### What happened?

<img width="1389" height="83" alt="Image" src="https://github.com/user-attachments/assets/52f8880d-8797-493e-b399-f25f6dd13efe" />
Same thing happened with ol, etc. it's just reporting the first key. Only way to make it work is to remove all options

**Comment by camc314** [comment]
hi! Thanks for reporting this - is this a regression?

I'll take a look
